dev-requirements.txt

```diff
@@ -9,6 +9,10 @@ pytest==4.6.9
 pytest-coverage
 coverage==4.5.4
 codecov==2.0.15
+PyYAML==5.3
+docker==4.1.0
+six==1.13.0
+pathlib2==2.3.5; python_version <= '2.7'
```

Comment:
I believe this conditional dependency notation only works with more recent version of setuptools so we need to ensure more recent / latest version is used everywhere where dev-requirements.txt is used.

This may or may not be an issue where the package builds run. Ideally we would eventually also run periodic builds on our Jenkins to ensure that pipeline is always working with the latest changes as well.

(Somewhat related - https://github.com/apache/libcloud/pull/1393/files#diff-2eeaed663bd0d25b7e608891384b7298R177)

Reply:
Maybe we just keep it until something shows up?)